src/main/resources/reference.conf

```diff
@@ -277,15 +277,21 @@ resourceTypes = {
       set_parent = {
         description = "set the parent resource of this google-project"
       }
+      add_child = {
+        description = "add child to google project"
+      }
     }
     ownerRoleName = "owner"
     roles = {
       owner = {
-        roleActions = ["read_policies", "list_notebook_cluster", "launch_notebook_cluster", "delete_notebook_cluster", "list_persistent_disk", "create_persistent_disk", "delete_persistent_disk", "create_kubernetes_app", "delete", "get_parent", "set_parent"]
+        roleActions = ["read_policies", "list_notebook_cluster", "delete_notebook_cluster", "list_persistent_disk", "delete_persistent_disk", "delete", "get_parent", "set_parent"]
         includedRoles = ["notebook-user"]
+        descendantRoles = {
+          kubernetes-app = ["manager"]
+        }
```

Comment:
this is the magic: the owner `role` of `google-project` confers the `manager` role on `kubernetes-app`s